core-web/libs/ui/src/lib/components/dot-drop-zone/dot-drop-zone.component.ts

```diff
@@ -31,15 +34,18 @@ export interface DropZoneFileValidity {
 export class DotDropZoneComponent {
     @Output() fileDropped = new EventEmitter<DropZoneFileEvent>();
     @Output() fileDragEnter = new EventEmitter<boolean>();
+    @Output() fileDragOver = new EventEmitter<boolean>();
     @Output() fileDragLeave = new EventEmitter<boolean>();
```

Comment:
can we unify this 3 in just one emitter? like `fileDragState`

Reply:
No, they were intended to be separate events. We had [Primeng](https://primeng.org/dragdrop) and [Angular cdk](https://material.angular.io/cdk/) as APi references.